src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -71,4 +71,8 @@ $toggle-switch-circle-size: $toggle-switch-height - 2 * $toggle-switch-spacing;
       margin: 8px 0 0;
     }
   }
+  
+  &:focus {
+    outline: 2px solid $color-dark-grey;
```

Comment:
Use `outline: 1px solid $color-axa-blue` like for the checkbox, to stay consistent.

Reply:
It says 2px in the [figma](https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=1765%3A888)